packages/algob/src/types.ts

```diff
@@ -347,6 +348,15 @@ export interface LsigInfo {
   lsig: Uint8Array
 }
 
+export interface DeployerConfig {
+  runtimeEnv: AlgobRuntimeEnv
+  cpData: CheckpointRepo
+  asaDefs: ASADefs
+  algoOp: AlgoOperator
+  accounts: Accounts
+  txWriter: txWriter
+}
```

Comment:
I think we can keep this type next to the `MkDeployerConfig` - user will not use it.

Reply:
Okay